tests/integration/test_fim/test_files/test_report_changes/data/test_cases/cases_large_changes.yaml

```diff
@@ -0,0 +1,155 @@
+- name: Test changes smaller than limit (Scheduled mode)
+  description: Test that changes are smaller than limit, 'More changes' does not appear in content_changes
+  configuration_parameters:
+    INTERVAL: 4
+    REALTIME: 'no'
+    WHODATA: 'no'
+  metadata:
+    filename: regular_1
+    original_size: 500
+    modified_size: 500
+    has_more_changes: false
+    fim_mode: scheduled
+
+- name: Test changes smaller than limit (Realtime mode)
+  description: Test that changes are smaller than limit, 'More changes' does not appear in content_changes
+  configuration_parameters:
+    INTERVAL: 10000
+    REALTIME: 'yes'
+    WHODATA: 'no'
+  metadata:
+    filename: regular_1
+    original_size: 500
+    modified_size: 500
+    has_more_changes: false
+    fim_mode: realtime
+
+- name: Test changes smaller than limit (Whodata mode)
+  description: Test that changes are smaller than limit, 'More changes' does not appear in content_changes
+  configuration_parameters:
+    INTERVAL: 10000
+    REALTIME: 'no'
+    WHODATA: 'yes'
+  metadata:
+    filename: regular_1
+    original_size: 500
+    modified_size: 500
+    has_more_changes: false
+    fim_mode: whodata
+
+- name: Test large changes - Same size (Scheduled mode)
+  description: Test that changes are bigger than limit, 'More changes' appears in content_changes
+  configuration_parameters:
+    INTERVAL: 4
+    REALTIME: 'no'
+    WHODATA: 'no'
+  metadata:
+    filename: regular_2
+    original_size: 200000
+    modified_size: 200000
+    has_more_changes: true
+    fim_mode: scheduled
+
+- name: Test large changes - Same size (Realtime mode)
+  description: Test that changes are bigger than limit, 'More changes' appears in content_changes
+  configuration_parameters:
+    INTERVAL: 10000
+    REALTIME: 'yes'
+    WHODATA: 'no'
+  metadata:
+    filename: regular_2
+    original_size: 200000
+    modified_size: 200000
+    has_more_changes: true
+    fim_mode: realtime
+
+- name: Test large changes - Same size (Whodata mode)
+  description: Test that changes are bigger than limit, 'More changes' appears in content_changes.
+  configuration_parameters:
+    INTERVAL: 10000
+    REALTIME: 'no'
+    WHODATA: 'yes'
+  metadata:
+    filename: regular_2
+    original_size: 200000
+    modified_size: 200000
+    has_more_changes: true
+    fim_mode: whodata
```

Comment:
Please, update the description.

Reply:
Done in https://github.com/wazuh/wazuh-qa/pull/4219/commits/d664f239c801fb219b9c529f5320e36c9508961b